### PR TITLE
[DC-3057] Run prototype to test retraction on a single dataset

### PR DESCRIPTION
**Pull request by nishanthpp93**
* [DC-3057] Add retraction prototype script

[DC-3057]: https://precisionmedicineinitiative.atlassian.net/browse/DC-3057?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: data_steward/tools/historical_retraction.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    dataset_query_job = bq_client.query(
+        f"SELECT * FROM {project_id}.{datasets_to_retract_dataset}.{datasets_to_retract_table}"
+    )
+    dataset_ids_result = dataset_query_job.result()
+    dataset_ids = dataset_ids_result.to_dataframe()["datasets"].to_list()
+    logging.info(f"Dataset id/s to target from table: {dataset_ids}")
+    logging.info(f"Running retraction on BQ datasets")
+
+    # retract from default dataset
+    # retract_data_bq.run_bq_retraction(project_id,
```
Comment (hiro-mishima) [inline data_steward/tools/historical_retraction.py:50]: Do you think we should uncomment this part?

**Review by hiro-mishima — COMMENTED** [review]
Thanks, @nishanthpp93, I left one comment. Let me know what you think. 

And do you think we should run the prototype for each data stage? (i.e., testing combined/ ehr/ unioned_ehr...)

File: data_steward/tools/historical_retraction.py (hunk 1 above)
Comment (nishanthpp93) [reply data_steward/tools/historical_retraction.py:50]: You are right, I missed this. Let me uncomment it.

**Review by hiro-mishima — APPROVED** [review]
Looks good! The CircleCi integration test failed, but it is irrelevant to this update.

**Pull request merged**